README.md

```diff
@@ -90,9 +88,9 @@ Below you can find the information on the custom options to set up the configura
 | _sampleRate_ | Defines the rate sampling. **It should be a number between [1, 100]**. | `number` | `100` | **NO** |
 | _tags_ | Object for setting the global tags. | `object` | `{}` | **NO** |
 | _transport_ | Defines the transport layer to be used to send metrics.<br><br> **Valid Transports:** `udp, api` | `string` | **none** | **YES** |
-| _host_ | Defines the hostname to where the metrics are sent. It can only be set inside _api_. | `string` | `127.0.0.1` | **NO** |
+| _host_ | Defines the hostname to where the metrics are sent. It can be set for _api_ or _udp_ whichever is applicable. | `string` | udp:   `127.0.0.1`<br>api: `'api.statful.com'` | **NO** |
 | _path_ | Defines the API path to where the metrics are sent. It can only be set inside _api_. | `string` | `/tel/v2.0/metric` | **NO** |
-| _port_ | Defines the port where the metrics are sent. It can only be set inside _api_. | `string` | `443` | **NO** |
+| _port_ | Defines the port where the metrics are sent. It can be set for _api_ or _udp_ whichever is applicable. | `string` | udp: `2013`<br>api: `443` | **NO** |
```

Comment:
Same as previous.

Here "Defines the port where the metrics are sent. It can only be set inside _api_.  Default values vary depending on transport value."